utils/createToken.js

```diff
@@ -0,0 +1,9 @@
+const jwt = require('jsonwebtoken');
+
+const generateToken = async (userId, name) => {
```

Comment:
What is `name`? The user doesn't have `name` field. It is better to encrypt `email` inside jwt token

Reply:
fixed